components/Product.js

```diff
@@ -10,9 +10,12 @@ import { ProductBody, ProductContainer } from '../styles.js';
 // TODO @tommypoa to use styled-components
 function Product({ product }) {
   return (
-    <View style={{ alignItems: 'center' }}>
+    <View style={{ alignItems: "center" }}>
       <Image
-        source={require('../assets/images/robot-dev.png')}
+        source={{
+          uri: product.image,
```

Comment:
Does this mean that the images are hosted on Airtable? I’m just wondering if there are any drawbacks to this :o

Reply:
It's for the sake of convenience when HC updates or adds new products since everything belongs in the Products table :0 but the tradeoff is that there's a bit of a loading time. In the future it would be good to load everything in one go when the clerk first logs in!